static/js/components/material/Drawer_test.js

```diff
@@ -89,25 +89,47 @@ describe("Drawer", () => {
     assert.isTrue(drawerNode.text().endsWith("Log out"))
   })
 
-  it("drawer element is rendered with collections list", async () => {
-    const wrapper = await renderDrawer()
-    ;[0, 1].forEach(function(col) {
-      const drawerNode = wrapper
-        .find(".mdc-list-item .mdc-list-item--activated")
-        .at(col)
-      assert.equal(drawerNode.text(), collections[col].title)
-      assert.equal(
-        drawerNode.props().href,
-        makeCollectionUrl(collections[col].key)
-      )
+  describe("when there are > 10 collections", () => {
+    beforeEach(() => {
+      const numCollections = 20
+      collections = [...Array(numCollections).keys()].map(() => makeCollection())
+      getCollectionsStub.returns(Promise.resolve({ results: collections }))
+    })
+
+    it("drawer element is rendered with max of 10 collections", async () => {
+      const wrapper = await renderDrawer()
+      const items = wrapper.find(".mdc-list-item .mdc-list-item--activated")
+      ;[0, 1, 3].forEach(function(col) {
+        const drawerNode = items.at(col)
+        assert.equal(drawerNode.text(), collections[col].title)
+        assert.equal(
+          drawerNode.props().href,
+          makeCollectionUrl(collections[col].key)
+        )
+      })
+    })
+
+    it("has 'more...' button that links to collections page", async () => {
+      const wrapper = await renderDrawer()
+      const moreButton = wrapper.find(".more-collections-button")
+      assert.equal(moreButton.length, 1)
+      assert.equal(moreButton.prop('href'), '/collections/')
     })
   })
 
-  it("drawer element is rendered with a help link", async () => {
-    const wrapper = await renderDrawer()
-    const drawerNode = wrapper.find(".mdc-list-item .mdc-link").at(1)
-    assert.equal(drawerNode.props().href, "/help/")
-    assert.isTrue(drawerNode.text().endsWith("Help"))
+  describe("when there are <= 10 collections", () => {
+    beforeEach(() => {
+      const numCollections = 10
+      collections = [...Array(numCollections).keys()].map(() => makeCollection())
+      getCollectionsStub.returns(Promise.resolve({ results: collections }))
+    })
+
+    it("does not have 'more...' button", async () => {
+      const wrapper = await renderDrawer()
+      const moreButton = wrapper.find(".more-collections-button")
+      assert.equal(moreButton.length, 0)
```

Comment:
I like doing `assert.isFalse(wrapper.find(".more-collections-button").exists())` for things like this, although it kind of amounts to the same thing :)